ensembl-track-api.openapi.yaml

```diff
@@ -61,18 +96,32 @@ paths:
           required: true
           description: Track ID
           schema:
-            type : string
+            type: string
+            format: uuid
           example: d0df738a-0ecb-4b1e-8576-a5621a4b15d2
       responses:
         '200':
           description: Successful request.
           content:
             application/json:
               schema:
-                type: object
-                $ref: '#/components/schemas/Track'
+                allOf:
+                  - $ref: '#/components/schemas/Track'
+                  - $ref: '#/components/schemas/TrackDatafiles'
+        '400':
+          description: Invalid request.
```

Comment:
How can this happen for this particular request?

Reply:
Unrecognized query params (`?xxxx`) could possibly the case, but not going to implement that so I'll remove this response.